domain/build.gradle

```diff
@@ -1,17 +1,66 @@
 plugins {
-    id 'java-library'
-    id 'org.jetbrains.kotlin.jvm'
+    id 'com.android.library'
```

Comment:
Why you import here android dependency? Main advantage of domain it was that there isn't android dependency here.